package.json

```diff
@@ -15,8 +15,9 @@
 	"scripts": {
 		"build": "remix vite:build",
 		"dev": "remix vite:dev",
-		"format": "prettier --write .",
+		"format": "prettier --cache --write .",
 		"lint": "eslint . .*js --max-warnings 0",
+		"lint:format": "prettier --cache --check .",
```

Comment:
The way I like to set up projects in general is `lint:*` scripts report you of violations and `format:*` scripts auto-fix said violations (as much as possible). I didn't set up this repo, but it seems to be done somewhat similarly. So here `lint:format` is a little contradicting to me.

Reply:
Agreed. Personally, I consider formatting to be a sibling to linting, not a sub-concern.

Reply:
Okay, that makes sense. I also did not notice significant speed increase, so I will close this.